code/modules/events/rescue_pod.dm

```diff
@@ -0,0 +1,31 @@
+/datum/event/rescue_pod
+    no_fake = 1
+    var/spawner_name = "rescuepodsurv"
+    var/datum/ghostspawner/human/rescuepodsurv/spawner
+
+/datum/event/rescue_pod/announce()
+    if(prob(66))
+        command_announcement.Announce("The NDV Icarus reports an escape pod inbound to the [station_name()]. Investigate and attend to the situation in accordance with NanoTrasen corporate regulations.", new_title="NDV Icarus")
```

Comment:
if possible this should have a soundclip - also maybe the burglar one shouldn't happen on extended

Reply:
the burglar one already doesn't happen on extended